### ISSUE TITLE
Help Center: Bring back message for incompatible browsers

### ISSUE THREAD

**Issue by kriskarkoski** [Automattic/wp-calypso#93430]
### Quick summary

Zendesk Messaging is incompatible with some browsers/configurations, notably Firefox private browsing and enhanced protection. We used to show a message in Odie in this situation to explain to the user what they needed to do to access chat. Previous to Omnichannel users could choose email, but that is no longer possible as a workaround. 

### Steps to reproduce

1. Open Help Center
2. Click Still Need Help
3. Try to connect to support through Wapuu

### What you expected to happen

To be able to submit a message to support through Help Center

### What actually happened

The widget shows an error and there is no warning message. Here is where it is in the code: 

https://github.com/Automattic/wp-calypso/blob/e3e0b326c06bcfb6541873e5f8a8abfabb777c1a/packages/help-center/src/components/help-center-contact-form.tsx#L545

The message is "In order to access the live chat widget, you will need to enable third-party cookies for WordPress.com." which we should update for our new experience. I think "In order to access support" would be flexible enough.

### Impact

Some (< 50%)

### Available workarounds?

Yes, difficult to implement

### If the above answer is "Yes...", outline the workaround.

Without a warning the user has to find our email or figure out they need to enable third-party cookies or switch browsers.

**Comment by retnonindya** [comment]
📌 REPRODUCTION RESULTS
Uncertain

📌 FINDINGS/SCREENSHOTS/VIDEO
I might did the process incorrectly 🤔 I tried with Firefox Private and Enhanced Protection, and it seems like I can submit question to Wapuu.

<img width="1355" alt="Screenshot 2024-08-14 at 3 39 45 PM" src="https://github.com/user-attachments/assets/3d83994a-12d9-4732-aa33-2da93a4b9972">

📌 ACTIONS
Triaged

**Comment by kriskarkoski** [comment]
Sorry if the steps were unclear @retnonindya, the issue is not with Wapuu but with contacting support (opening the Zendesk widget) after talking to Wapuu.

**Comment by kriskarkoski** [comment]
I suspect #93427 is related to this. 1Blocker grays out the contact support option (p1723724531557079-slack-CPJE78JQ5), presumably because the widget is blocked and the `isMessagingScriptLoaded` check mentioned on that issue fails. In addition to bringing back a message explaining how to access Messaging we could consider showing the email fallback option in these cases if uses don't care/want to deal with adjusting settings or switching browsers.

**Comment by pauljacobson** [comment]
We received a report about this in 8699495-zen where the user uses Firefox v130 on Windows 10 ([system info](https://www.whatsmybrowser.org/b/K1D8F)). The user tested this in a Private window and shared these console logs: 359da-pb/

The user added the following:

> I can confirm that we are using Firefox 130.0 (64-bit), have allowed wordpress.com in Privacy settings on the browser, and have disabled adblocking extensions for your domain.

**Comment by arcangelini** [comment]
I spent some time testing this out and was unable to reproduce this error. I tested in seemingly the same environment as the user in the ticket shared. All attempts to start a chat ended with me seeing the contact button greyed out and the notice to enable cookies on my computer.

I tried with multiple different security settings and in private mode. All led to the same result. All attempts to get the widget to load were fruitless. Is anyone else able to replicate this?

![Image](https://github.com/user-attachments/assets/324fdb26-25f4-420b-90db-2830dfae189d)
